libs/sdk/src/client.ts

```diff
@@ -992,6 +992,28 @@ export class ThreadsClient<
       }
     );
   }
+
+  async *getStream(
+    threadId: string,
+    lastEventId?: string
```

Comment:
_nit_: make this accept an object instead for future proofing

```suggestion
    options?: { lastEventId?: string }
```